### ISSUE TITLE
Heavy Loaded

### ISSUE THREAD

**Issue by androidapkr** [mukeshsolanki/photofilter#2]
First of thanks for doing wonderful work on image processing.

As my personal experience i had checked this processing this is heavy loaded on memory.
Big Issue : Memory Management

**Comment by Seddiks** [comment]
Yeah i agree with you, there are many filters and thats cool but processing is very slow.. i dont think its problem of memory, but it make processor work a lot!

**Comment by xeroxstar** [comment]
To solve it, I resize the image before editing it, the process on small images is much better than on original size. Anyway, thanks for this plugin, works great.
